composer.json

```diff
@@ -33,14 +34,13 @@
         "phpunit/phpunit": "^9.5",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "sensiolabs/security-checker": "^6.0",
-        "sylius-labs/coding-standard": "^4.0",
         "symfony/browser-kit": "^4.4 || ^5.2",
         "symfony/debug-bundle": "^4.4 || ^5.2",
         "symfony/dotenv": "^4.4 || ^5.2",
         "symfony/intl": "^4.4 || ^5.2",
         "symfony/web-profiler-bundle": "^4.4 || ^5.2",
-        "symplify/easy-coding-standard": "^8.3",
-        "vimeo/psalm": "4.6.4"
+        "symplify/easy-coding-standard": "^9.4",
+        "vimeo/psalm": "4.24.0"
```

Comment:
Why this changes?

Reply:
These packages had too low versions and the update was required during installation of "bitbag/coding-standard".